doc/articles/uno-development/Uno-UI-XAML-ResourceTrimming.md

```diff
@@ -12,17 +12,37 @@ In order to determine what is actually used by the application, the Uno Platform
 - The tooling also embeds an ILLinker substitution file allowing the linker to unconditionally remove the code that conditionally references those properties. For instance, for `LinkerHints.Is_Windows_UI_Xaml_Controls_Border_Available`, any block of `if (LinkerHints.Is_Windows_UI_Xaml_Controls_Border_Available)` will be removed when the `--feature Is_Windows_UI_Xaml_Controls_Border_Available false` parameter is provided to the linker.
 
 Then the multiple passes of IL Linker are done:
-- The first pass runs the IL Linker with all XAML resources and Binding Metadata disabled by setting all `LinkerHints` properties to false. This removes all code directly associated to those Bindable Metadata and XAML Resources. This has the effect of only keeping framework code which is directly referenced from user code.
+- The first pass runs the IL Linker with all XAML resources and Binding Metadata disabled by setting all `__LinkerHints` properties to false. This removes all code directly associated to those Bindable Metadata and XAML Resources. This has the effect of only keeping framework code which is directly referenced from user code.
 - The tooling then reads the result of the linker to determine which types in the `LinkerHints` are still available in the assemblies.
-- The subsequent passes run the IL Linker with `LinkerHints` enabled only for types detected to be used during the first pass. This will enable types indirectly referenced by XAML Resources (e.g. a ScrollBar inside a ScrollViewer style) to be kept by the linker.
-- The tooling then reads again the result of the linker to determine which types in the `LinkerHints` are still available in the assemblies.
+- The subsequent passes run the IL Linker with `__LinkerHints` enabled only for types detected to be used during the first pass. This will enable types indirectly referenced by XAML Resources (e.g. a ScrollBar inside a ScrollViewer style) to be kept by the linker.
+- The tooling then reads again the result of the linker to determine which types in the `__LinkerHints` are still available in the assemblies.
 - The tooling re-runs this last pass until the available types list stops changing.
 
-The resulting `LinkerHints` types are now passed as features to the final linker pass (the one bundled with the .NET tooling) to generate the final binary, containing only the used types.
+The resulting `__LinkerHints` types are now passed as features to the final linker pass (the one bundled with the .NET tooling) to generate the final binary, containing only the used types.
 
 As of Uno 3.9, the Uno.UI WebAssembly assembly is 7.5MB, trimmed down to 3.1MB for a `dotnet new unoapp` template.
+
+## Using the `AdditionalLinkerHintAttribute` attribute
+
+In some scenarios (e.g. ExpandoObject), it may be needed to generate linker hints for non-DependencyObject types.
+
+To get a `__LinkerHints` property for an additional type, add the following in AssemblyInfo.cs:
```

Comment:
```suggestion
To get a `__LinkerHints` property for an additional type, add the following in AssemblyInfo.cs:

```

Reply:
Note that this adheres to markdownlint rules. I'm hoping to early catch markdownlint violations so that it is easier in the future if we enabled markdownlint in CI. The same applies to below comment as well.